### DISCUSSION TITLE
I understand it is deprecated, but do we have a schedule to completely inactivate Nodejs 16 binary?

### DISCUSSION THREAD

**Discussion by FeidaMeng** [nodesource/distributions#1777] [General]
Our NodeJS is still on version 16, would like to schedule an upgrade based on the answer.

Thank you very much.

**Comment by riosje** [comment]
Hi @FeidaMeng node16 has not support anymore, has reached the EOL the last year, highly suggest you to upgrade to Node20 
![image](https://github.com/nodesource/distributions/assets/55195249/6c13e0ad-2b78-4c63-b57b-ae41564f0e75)
https://nodejs.org/en/about/previous-releases

**Reply by FeidaMeng** [reply]
thanks for the quick response!
